pytransform3d/test/test_transform_manager.py

```diff
@@ -504,6 +504,47 @@ def test_numpy_timeseries_transform():
     assert origin_of_A_in_B_y == pytest.approx(-1.28, abs=1e-2)
 
 
+def test_numpy_timeseries_transform_multiple_timestamps_query():
```

Comment:
Since `pytest` is used, consider extending the existing tests using `pytest.mark.parametrize`:

```python
@pytest.mark.paramertrize("query_times", [4.9, np.array([4.9, 4.9])]
def test_numpy_timeseries_transform(query_times):
    # ...
    tm.get_transform_at_time("A", "B", query_times)
```

Reply:
Sounds like a good idea. Can we do this?